roachtest: exercise drop of 2tb table

tbg opened the pull request:
This roachtest replicates the scenario described in #24029 and serves
as the basis for experimentation.

It currently fails:

```
[  13] clearrange: repeatedly running COUNT(*) on small table (5.0%|46s)
read 800 rows
[  14] clearrange: repeatedly running COUNT(*) on small table (10.0%|1m46s)
read 800 rows
[  15] clearrange: repeatedly running COUNT(*) on small table (15.0%|2m46s)
read 800 rows
[  16] clearrange: repeatedly running COUNT(*) on small table (20.0%|3m46s)
read 800 rows
[  17] clearrange: repeatedly running COUNT(*) on small table (25.0%|4m46s)
read 800 rows
[  18] clearrange: repeatedly running COUNT(*) on small table (30.0%|5m46s)
read 800 rows
[  19] clearrange: repeatedly running COUNT(*) on small table (35.0%|6m46s)
read 800 rows
[  20] clearrange: repeatedly running COUNT(*) on small table (40.0%|7m46s)
read 800 rows
[  21] clearrange: repeatedly running COUNT(*) on small table (45.0%|8m46s)
[  22] clearrange: repeatedly running COUNT(*) on small table (45.0%|9m46s)
[  23] clearrange: repeatedly running COUNT(*) on small table (45.0%|10m46s)
[  24] clearrange: repeatedly running COUNT(*) on small table (45.0%|11m46s)
read 800 rows
read 800 rows
[  25] clearrange: repeatedly running COUNT(*) on small table (55.0%|12m46s)
read 800 rows
[  26] clearrange: repeatedly running COUNT(*) on small table (60.0%|13m46s)
read 800 rows
[  27] clearrange: repeatedly running COUNT(*) on small table (65.0%|14m46s)
read 800 rows
[  28] clearrange: repeatedly running COUNT(*) on small table (70.0%|15m46s)
read 800 rows
[  29] clearrange: repeatedly running COUNT(*) on small table (75.0%|16m46s)
[  30] clearrange: repeatedly running COUNT(*) on small table (75.0%|17m46s)
[  31] clearrange: repeatedly running COUNT(*) on small table (75.0%|18m46s)
[  32] clearrange: repeatedly running COUNT(*) on small table (75.0%|19m46s)
[  33] clearrange: repeatedly running COUNT(*) on small table (75.0%|20m46s)
[  34] clearrange: repeatedly running COUNT(*) on small table (75.0%|21m46s)
[  35] clearrange: repeatedly running COUNT(*) on small table (75.0%|22m46s)
[  36] clearrange: repeatedly running COUNT(*) on small table (75.0%|23m46s)
[  37] clearrange: repeatedly running COUNT(*) on small table (75.0%|24m46s)
[  38] clearrange: repeatedly running COUNT(*) on small table (75.0%|25m46s)
[  39] clearrange: repeatedly running COUNT(*) on small table (75.0%|26m46s)
[  40] clearrange: repeatedly running COUNT(*) on small table (75.0%|27m46s)
[  41] clearrange: repeatedly running COUNT(*) on small table (75.0%|28m46s)
[  42] clearrange: repeatedly running COUNT(*) on small table (75.0%|29m46s)
[  43] clearrange: repeatedly running COUNT(*) on small table (75.0%|30m46s)
[  44] clearrange: repeatedly running COUNT(*) on small table (75.0%|31m46s)
[  45] clearrange: repeatedly running COUNT(*) on small table (75.0%|32m46s)
[  46] clearrange: repeatedly running COUNT(*) on small table (75.0%|33m46s)
[  47] clearrange: repeatedly running COUNT(*) on small table (75.0%|34m46s)
[  48] clearrange: repeatedly running COUNT(*) on small table (75.0%|35m46s)
...
```

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/25837)
<!-- Reviewable:end -->

petermattis commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 0 of 2 files reviewed at latest revision, all discussions resolved.

---

*[pkg/cmd/roachtest/clearrange.go, line 33 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/25837#-LDCnJlD2mJQxQ8xjQXV:-LDCnJlD2mJQxQ8xjQXW:b-hwhxsl) ([raw file](https://github.com/cockroachdb/cockroach/blob/36e5412a18a8780d6d880cbf79207bd9a2b3426a/pkg/cmd/roachtest/clearrange.go#L33)):*
> ```Go
> 		Stable: false,
> 		Run: func(ctx context.Context, t *test, c *cluster) {
> 			t.Status(`downloading store dumps`)
> ```

I could use a bit more commentary in this test. It took me a few reads to understand that the store dumps are large, while the import into tinybank is small. 

---


*Comments from [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/25837#-:-LDCnZaA5HzAnm4d-qlM:bnfp4nl)*
<!-- Sent from Reviewable.io -->

petermattis commented:
Do you plan to merge this? Perhaps mark it as skipped.

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/25837#-:-LDNJ7Ux9kPRD4ZDpOFI:bgrq9ct)*
<!-- Sent from Reviewable.io -->

tbg committed:
roachtest: exercise drop of 2tb table

This roachtest replicates the scenario described in #24029 and serves
as the basis for experimentation.

It currently fails.

Release note: None

tschottdorf commented:
TFTR! I'll merge this (and I'll leave it unskipped for a few days, just to see that it reliably fails and that I didn't introduce a bug in the home stretch).

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[pkg/cmd/roachtest/clearrange.go, line 33 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/25837#-LDCnJlD2mJQxQ8xjQXV:-LD_7eKP51VfGmpcMfEv:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/36e5412a18a8780d6d880cbf79207bd9a2b3426a/pkg/cmd/roachtest/clearrange.go#L33)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

I could use a bit more commentary in this test. It took me a few reads to understand that the store dumps are large, while the import into tinybank is small. 
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/25837#-:-LD_7y2N3AW_30tupbzs:bjm6x7c)*
<!-- Sent from Reviewable.io -->

tschottdorf commented:
bors r=petermattis
